viz/tap/cmd/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/linkerd/linkerd2/viz/tap/controller"
+	"github.com/linkerd/linkerd2/viz/tap/injector"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("expected a subcommand")
+		os.Exit(1)
+	}
+	switch os.Args[1] {
+	case "controller":
```

Comment:
Could this naming be confusing? I understand that now tap is like an umbrella concept, in which injector and the API are part of. Should this be `tap-api` or `api` instead? 🤔 

My main concern is this does not perform usual characteristics like that of a controller right? i.e reconciling based on events.

Reply:
Yea good call. I renamed it to `api`. `tap/api` now contains all the Go files for the gRPC and API servers.

Reply:
I've been known to say "tap controller" when talking about this.  But I agree with @Pothulapati that calling this "api" is probably more specific.